keyboards/gray_studio/hb85/hb85.h

```diff
@@ -1,5 +1,5 @@
 /*
-Copyright 2017 Luiz Ribeiro <luizribeiro@gmail.com>
+Copyright 2019 MechMerlin <mechmerlin@gmail.com>
```

Comment:
Could we have both listed?

Reply:
Actually good catch, Luiz is the original author of the first bmc->qmk ports. That should actually be Felipe Coury.